examples/sorted-set.js

```diff
@@ -25,9 +27,23 @@ async function addToSortedSet() {
   // Get all of the values/scores from the sorted set using
   // the scan approach:
   // https://redis.io/commands/zscan
+  console.log('Using ZCSCAN:');
   for await (const memberWithScore of client.zScanIterator('mysortedset')) {
     console.log(memberWithScore);
   }
+
+  // Get all of the values/scores from the sorted set using
```

Comment:
I think that we should add a note that this is not a good idea..

Reply:
Probably yes, can we do this with `ZSCAN` with `WITHSCORES` and `REV` modifiers?

Reply:
`ZSCAN` gives the score (so there is no need for the `WITHSCORES` modifier), but does not support the `REV` modifier